website/content/gateway/api_reference/changes/changes.md

```diff
@@ -10,6 +10,17 @@ Follow coming changes on the [source code repository](https://github.com/clearha
 
 Sorted by descending timestamp.
 
+#### Removal of deprecated parameters
+We are removing several parameters that have been deprecated for an extended
+period of time. We promise that the following parameters are functional until
+2023-05-01, but give no guarantees thereafter.
```

Comment:
```suggestion
2023-05-01T00:00:00Z but give no guarantees thereafter.
```

:man_shrugging:

Reply:
I do get the point, however, we won't be killing it right after the deadline, but likely only the next day (or even a few days) later — so the precision is not too important. On the other hand, it doesn't do any harm, so I'll take it — to also get rid of the comma 🙂